docs/guides/styling.md

```diff
@@ -448,6 +448,7 @@ Finally, import the generated CSS file into your app:
 
 ```tsx filename=app/root.tsx
 // ...
+import type { LinksFunction } from "@remix-run/node";
```

Comment:
I think it would be better to do it like this then

```suggestion
import type { LinksFunction } from "@remix-run/node"; // or "@remix-run/cloudflare"

// ...

```